root/var/www/html/config.inc.php

```diff
@@ -4,38 +4,38 @@
  * Database settings
  ******************/
 // Which database system: "pgsql"=PostgreSQL, "mysql"=MySQL
-$dbsys = "mysql";
+$dbsys = $_ENV['MRBS_DB_SYSTEM'];
```

Comment:
`$_ENV` seems to be empty and does not to work without additional configuration. Does it work for you?

Reply:
Can you try with the latest version, please

Reply:
Still doesn't work. I added
```php
echo "ENV: '". $_ENV['MRBS_DB_SYSTEM'] . "', getenv: '" . getenv('MRBS_DB_SYSTEM') . "'\n";
```
To my `config.inc.php` and I get this result:
![image](https://github.com/dorianim/mrbs-docker/assets/30153207/eb6355af-693e-4d1b-b1a8-fe919eb4e321)

Reply:
Ok, thanks. So getenv() seems to be more reliable. Ok, I change the calls to use getenv()